src/riak_dt_pb.erl

```diff
@@ -4145,6 +4145,7 @@ verify_msg(Msg, Opts) ->
     end.
 
 
+-dialyzer({nowarn_function,v_msg_mapfield/3}).
```

Comment:
These will have to go in the `dialyzer.ignore-warnings` file I think, probably a syntax that was added in R16 or R17.